### DISCUSSION TITLE
Mixed Content: The page at 'https://...' was loaded over HTTPS, but requested an insecure resource ... 문제 해결

### DISCUSSION THREAD

**Discussion by jinyoung234** [dnd-side-project/dnd-8th-1-frontend#170] [00. Trouble Shooting]
# 문제 원인
![image](https://user-images.githubusercontent.com/87177577/219955859-09eb21ab-6bf6-4b24-a940-a2c7c8a58fed.png)

암호화된 HTTPS 페이지에 암호화되지 않은 HTTP를 통해 요청할 때 발생하는 에러이다. 프론트는 https로 설정되어 있지만 백엔드의 경우 http로 설정되어있기 때문에 해당 에러가 발생한 것이다.

# 해결 방법
<img width="413" alt="image" src="https://user-images.githubusercontent.com/87177577/219955946-070e682a-781d-459d-abcd-98f1c763c2eb.png">

_document.tsx에 `Content-Security-Policy` 메타태그를 추가하면 더 이상 에러가 발생하지 않는 것을 확인할 수 있다.

<img width="996" alt="image" src="https://user-images.githubusercontent.com/87177577/219955993-c54f5b18-6af9-4f19-bb0c-246dc11cffd9.png">
하지만 `net::err_connection_timed_out`가 발생하며 이 원인은 다시 찾아봐야 할 거 같다.

## 레퍼런스
[mixed-content-error](https://velog.io/@nemo/mixed-content-error)

**Comment by punchdrunkard** [comment]
서버문제임
